cms/templates/widgets/footer.html

```diff
@@ -1,14 +1,19 @@
+<%page expression_filter="h"/>
+
 <%!
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
+from openedx.core.djangolib.markup import HTML, Text
 %>
 
+<%namespace name='static' file='../static_content.html'/>
```

Comment:
Why did you make this change? I'm not saying it is wrong, just trying to understand as marketing_link seemed to work before (and it actually comes from edxmako.shortcuts).